docs/.vitepress/config.ts

```diff
@@ -0,0 +1,118 @@
+import { defineConfig } from 'vitepress'
+
+const nav = [
+  { text: 'Getting started', link: '/presets' },
+  { text: 'Guide', link: 'preset-uno' },
+  { text: 'Reference', link: '/interactive' },
+  { text: 'Playground', link: '/play' },
```

Comment:
Reference and playground should be external links to the corresponding sites

Reply:
I was about to add the following links but @antfu mentioned the `unocss.dev` domain that I didn’t know was available.

``` js
{ text: 'Reference', link: 'https://uno.antfu.me/reference/' },
{ text: 'Playground', link: 'https://uno.antfu.me/play/' },
```

Regarding the interactive docs, what’s the preferred route? `interactive` or `reference`?

Reply:
The domain is for the new docs deployed on netlify, the links to include are:
- Interactive Docs (should change Reference): https://uno.antfu.me/
- Playground: https://uno.antfu.me/play/

Reply:
Thanks. Done: https://github.com/unocss/unocss/pull/2386/commits/40908aaf86d27027be2571b912a6aa68a963fe46

Reply:
I will move them to the new domain, no worries

Reply:
can you replace `Reference` with `Interactive Docs`?

Reply:
Sure! https://github.com/unocss/unocss/pull/2386/commits/943cb492f7c57a98540d26a0f1d935c346f94ce4